modules/auto-delete/events/voiceStateUpdate.js

```diff
@@ -0,0 +1,25 @@
+const {localize} = require('../../../src/functions/localize');
+module.exports.run = async function (client, oldState) {
+    if (!client.botReadyAt) return;
+
+    const voiceChannels = client.configurations['auto-delete']['voice-channels'];
+
+    const channelConfigEntry = voiceChannels.find((vc) => oldState.channelId === vc.channelID);
+    if (!channelConfigEntry) return;
+
+    const channel = await client.channels.fetch(channelConfigEntry.channelID).catch(() => {});
+    if (!channel) {
+        return client.logger.error(`[auto-delete] ${localize('auto-delete', 'could-not-fetch-channel', {c: channelConfigEntry.channelID})}`);
+    }
+    if (channel.members.size === 0) return;
```

Comment:
You are returing here, if the channel is empty. As you want to delete the messages if the channel is empty, this statment needs to be switched.

```suggestion
    if (channel.members.size !== 0) return;
```

Reply:
i used a different approach to make it more readable

Reply:
Yeah, but your approch only if the channel has members in it, not if it's empty?

Reply:
You are existing the code executing if the channel has no members in it. This will result in the opposite of the intended result. You want to stop execution if the channel is not empty, not if it's empty. While readability is great, the whole feature does not work as intended, if this if-statement is empty

Reply:
You are absolutly right. I used these Conversations as a check list. 
I used: following: ` if (channel.members.size !== 0) return;`
Tbh didnt expect you to answer that 😄

Reply:
oh sorry 😅